cmd/kgctl/connect_linux.nogo

```diff
@@ -0,0 +1,473 @@
+// +build linux
```

Comment:
This is supposed to be in another pull request, right?

Reply:
yes